modules/serverless-ossm-secret-filtering-net-istio.adoc

```diff
@@ -25,30 +25,27 @@ endif::[]
 
 .Procedure
 
-* Add the `serverless.openshift.io/enable-secret-informer-filtering` annotation to the `KnativeServing` CR:
```

Comment:
According to SRVKS-947 this annotation is going to be deprecated in 1.28 not removed. 
We need to follow the RH deprecation policy. Plan I guess is to remove it completely in 1.29+.

Reply:
Users need to annotate their secrets if they use this feature with:
`    networking.internal.knative.dev/certificate-uid: "some_id". Do we capture this somewhere?
They need to start doing this anyway because at some feature release we might enable this by default.
The idea was to enable the feature by default on [1.29](https://github.com/openshift-knative/serverless-operator/pull/1954#issue-1569812367) but probably it will take longer due to doc changes need.

Reply:
That is good, thanks. We should also reference in docs that if users enable this feature they should annotate their secrets (including existing ones) with: ` networking.internal.knative.dev/certificate-uid: "some_id", otherwise Serving will not "see" them and there will be failures.

Reply:
But, as the annotation is deprecated now, shouldn't our docs now instead recommend using the  ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID env instead even for net-istio?

Reply:
@skonto Yeah, we captured the need to annotate secrets in the release notes. However, I added two notes into the net-istio and net-kourier sections as well.

@skonto @maschmid So should I go with the annotation or env var?

Reply:
@maschmid I've looked again at the docs provided by Stavros, and looks like it should be annotation for net-istio. If this ends up being the worse option, LMK  and I'll hotfix this step.